SYCL/ESIMD/api/functional/ctors/ctor_fill_core.cpp

```diff
@@ -21,6 +21,10 @@
 //  - construct simd with pre-defined base and step value
 //  - bitwise comparing expected and retrieved values
 
+#ifdef ESIMD_TESTS_FULL_COVERAGE
+#define ESIMD_TESTS_FULL_COVERAGE
+#endif
+
```

Comment:
```suggestion
```

Reply:
applied in this commit: 02c57e16c20cd1583fc2257fd839d0de2387eaa8